aiven/client/cli.py

```diff
@@ -1055,12 +1070,12 @@ def service__privatelink__aws__delete(self):
     @arg.service_name
     @arg("--format", help="Format string for output")
     @arg.json
-    def service__privatelink__aws__connection__list(self):
+    def service__privatelink__aws__connection__list(self) -> None:
         """List PrivateLink connections for a service"""
         resp = self.client.list_service_privatelink_aws_connections(
             project=self.get_project(), service=self.args.service_name
         )
-        self.print_response([resp], format=self.args.format, json=self.args.json)
+        self.print_response(resp, format=self.args.format, json=self.args.json)
```

Comment:
because `list_service_privatelink_aws_connections` is actually returning a `Sequence`, not a `Mapping`. And in `print_response`, it's annotated that it accept `Union[Mapping[str, Any], Collection[Mapping[str, Any]]]`

Reply:
To add more information, with the old way of `[resp]` the result was printed as a dict, without proper table format like this 
```
{"dns_name": "some-user-name.aws.com", "privatelink_connection_id": "plc3abd778b14d", "state": "active", "vpc_endpoint_id": "vpce-01234567abcdef"}
```

Now we fix, it would print the table like this:
```
DNS_NAME                PRIVATELINK_CONNECTION_ID  STATE   VPC_ENDPOINT_ID    
======================  =========================  ======  ===================
some-user-name.aws.com  plc3abd778b14d             active  vpce-01234567abcdef
```